proteus/NonlinearSolvers.py

```diff
@@ -472,7 +472,7 @@ def setLinearSolverTolerance(self,r):
         self.norm_r_last = self.norm_r
         self.linearSolver.setResTol(rtol=eta,atol=self.linearSolver.atol_r)
     def solve(self,u,r=None,b=None,par_u=None,par_r=None,linear=False):
-        r""" Solves the non-linear system :math:`F(u) = b`.
+        """ Solves the non-linear system :math:`F(u) = b`.
```

Comment:
was this just a typo or something related to formatting?

Reply:
I removed the "r" at the beginning of the comment, I assumed it was a typo.

Reply:
The "r" before the string signifies that that it's a raw string: https://docs.python.org/3/reference/lexical_analysis.html. That means you get almost exactly what is between the """'s, otherwise Python will interepret escape sequences. Mainly important when you include latex commands in the string.